models/issues/issue.go

```diff
@@ -1335,14 +1342,24 @@ func (opts *IssuesOptions) setupSessionNoLimit(sess *xorm.Session) {
 	}
 
 	if opts.LabelIDs != nil {
+		var noLabelsIDs []int64
+		hasNoLabel := false
 		for i, labelID := range opts.LabelIDs {
 			if labelID > 0 {
 				sess.Join("INNER", fmt.Sprintf("issue_label il%d", i),
 					fmt.Sprintf("issue.id = il%[1]d.issue_id AND il%[1]d.label_id = %[2]d", i, labelID))
-			} else {
-				sess.Where("issue.id not in (select issue_id from issue_label where label_id = ?)", -labelID)
+			} else if labelID < 0 {
+				// sess.Where("issue.id not in (select issue_id from issue_label where label_id = ?)", -labelID)
+				noLabelsIDs = append(noLabelsIDs, -labelID)
+				hasNoLabel = true
 			}
 		}
+		if hasNoLabel {
+			sess.NotIn("issue.id",
```

Comment:
```go
sess = sess.And(builder.NotExists(
      builder.Select("id").From("issue_label").
      Where(builder.Expr("issue_label.issue_id = issue.id"),
))
break
```

Reply:
This modify in label using alt key is not work